lib/tpm2_options.c

```diff
@@ -267,35 +269,24 @@ static void show_version (const char *name) {
 
 void tpm2_print_usage(const char *command, struct tpm2_options *tool_opts) {
     unsigned int i;
-    bool indent = true;
-    char *command_copy;
 
-    if (!tool_opts || !(tool_opts->flags & TPM2_OPTIONS_SHOW_USAGE)) {
+    if (!tool_opts || ( !(tool_opts->flags & TPM2_OPTIONS_SHOW_USAGE) & !tpm_force_show_usage )) {
         return;
     }
 
-    command_copy = strdup(command);
-    printf("Usage: %s%s%s\n", basename(command_copy),
-           tool_opts->callbacks.on_opt ? " [<options>]" : "",
-           tool_opts->callbacks.on_arg ? " <arguments>" : "");
-    free(command_copy);
+    printf("usage: %s%s%s\n", command,
+           tool_opts->callbacks.on_opt ? " [OPTIONS]" : "",
+           tool_opts->callbacks.on_arg ? " ARGUMENTS" : "");
```

Comment:
Why did this change? Is this the proper output syntax?

Reply:
thanks for the review, i didn't changue that, maybe the branch wasn't update, i'll fix that